src/module/entities/helper.js

```diff
@@ -1274,7 +1274,7 @@ module.exports = class UserProjectsHelper {
 				// Find the entities document based on the entityType in queryParams
 
 				let tenantId = userDetails.tenantAndOrgInfo.tenantId
-				let orgId = CONSTANTS.common.ALL
+				let orgId = userDetails.tenantAndOrgInfo.orgId[0]
```

Comment:
@borkarsaish65  why orgId[0]? Is the same user can belong multiple organization ?

Reply:
Yes but @aks30 had told we have to take the first element. This is just to seggregate the data  according to Saas Policy.